### ISSUE TITLE
 WARN  Legacy styleResources detected. Will take over but ignore all other rules. Please move the rules to the top-level styleResources key

### ISSUE THREAD

**Issue by Innei** [nuxt-community/style-resources-module#88]
I followed the tutorial to setup the module.

and the following warning occurred.

```js
 WARN  Legacy styleResources detected. Will take over but ignore all other rules. Please move the rules to the top-level styleResources key
```

this is my `nuxt.config.js` snippet.
```js
 css: ['~/assets/css/normalize.css'],
 modules: [
    '@nuxtjs/style-resources',
    // Doc: https://axios.nuxtjs.org/usage
    '@nuxtjs/axios',
    // Doc: https://github.com/nuxt-community/dotenv-module
    '@nuxtjs/dotenv'
  ],
 build: {
    transpile: [/^element-ui/],
    styleResources: {
      scss: '~assets/scss/_color.scss'
    },
}
```

How can I resolve this warning.

**Comment by uran1980** [comment]
I have the same issue. @Innei how did you solve this? 
Аnd why did you close this issue without any description how to solve it?

**Comment by manniL** [comment]
@uran1980 the error already hints you how to solve it. move `build.styleResources` to a top level key of the nuxt.config.js object.

```js
{
  styleResources: { /*...*/ }
}
```
